pkg/api/customization/globalnamespaceaccess/access_common.go

```diff
@@ -86,6 +86,45 @@ func (ma *MemberAccess) IsAdmin(callerID string) (bool, error) {
 	return false, nil
 }
 
+func (ma *MemberAccess) CanCreateRKETemplate(callerID string) (bool, error) {
```

Comment:
This can be simplified by doing something similar to https://github.com/rancher/rancher/blob/master/pkg/auth/requests/sar/sar.go#L52. This uses the k8s auth check.